docs/creating_component_code_review.md

```diff
@@ -49,7 +49,7 @@ Not all existing code follows the requirements in this checklist. This cannot be
     status = bridge.status()
     ```
 
-    [Tutorial on publishing your own PyPI package](https://jeffknupp.com/blog/2013/08/16/open-sourcing-a-python-project-the-right-way/)
+    [Tutorial on publishing your own PyPI package](https://web.archive.org/web/20201214181824/https://jeffknupp.com/blog/2013/08/16/open-sourcing-a-python-project-the-right-way/)
```

Comment:
This tutorial is very outdated and doesn't represent the current best practices, eg its parameters in setup.py and way to upload to PyPI.

I found this newer more simple guide:
https://towardsdatascience.com/how-to-open-source-your-first-python-package-e717444e1da0

It's not complete and still has a link to Travis at the end but I think it's better than the old guide. We could consider linking separately to the cookiecutter project also.

Reply:
This guide is more comprehensive and follow many modern practices. It's a bit long though including all the chapters 1-6:
https://cjolowicz.github.io/posts/hypermodern-python-01-setup/

Reply:
Other noteworthy projects could possibly be flit and poetry.

Reply:
Or other articles like:
https://medium.com/developer-circle-kampala/grow-your-python-how-i-published-my-first-python-package-to-pypi-162c3c709349
or
http://brunorocha.org/python/publish-your-python-packages-easily-using-flit.html